_resources/production_liasion_guidelines.md

```diff
@@ -0,0 +1,125 @@
+
+```bash
+ssh <username>@osgsub01 -Y
+```
+
+Report any difficulties to the above-mentioned email for filing tickets. 
+
+---
+# Getting Write Access to Jefferson Lab Rucio Storage Endpoint (RSE)
+### Getting a Certificate and Key
+
+1. If you have a jlab account, file a ticket with [Service Now](https://jlab.servicenowservices.com/) asking for a certificate to be able access ePIC Rucio in the context of OSG job submission. If you don't have a jlab account, contact the [production WG](https://eic.github.io/epic-prod/about/contact.html) to file a ticket on your behalf.
+
+2. You will receive an email from support@cert-manager.com with the instruction to download your .cer file.
+​
+3. Once you have received this .cer file, transfer this to the access point you will be using. Please request the production WG to provide you with the key corresponding to your .cer file. Your key will be placed in the access point according to your request (BNL, JLAB or OSG). 
+​
+4. Move certificates and keys to a desired location and change the permissions.
+    ```bash
+    chmod 600 usercert.cer
+    chmod 600 userkey.key
+    ```
+
+### Installing voms-client
+
+You will need the `voms-client` on a Linux machine. If you are on your local Linux machine, then do:
+
+```bash
+sudo apt-get install voms-clients-java
+```
+All the access points should already have the `voms-client` installed. So, you can skip this step if you are already on one. 
+
+### Running voms-proxy-init
+
+Run the following command to generate a voms proxy certificate with a validity of at least 2 months (1460 hours):
+
+```bash
+voms-proxy-init -cert=usercert.cer -key=userkey.key -out=x509_user_proxy -hours=1460
+```
+
+This will create an proxy certificate in your current directory. 
+
+### Send Information
+
+Run the following command and email the output to `panta@jlab.org`:
+
+```bash
+voms-proxy-info -file=x509_user_proxy | grep "issuer" | awk -F":" '{print $2}'
+```
+You only need to do this once. 
+
+---
+# Integrating a New Dataset in Production
+
+Run eic-shell and set the rucio variables
+```bash
+./eic-shell
+export RUCIO_CONFIG=/opt/campaigns/hepmc3/scripts/rucio.cfg
+export X509_USER_PROXY=x509_user_proxy
+```
+
+Then you can transfer the files from the source location to desired directory structure on JLAB RSE following the [input pre-processing guidelines](https://eic.github.io/epic-prod/documentation/input_preprocessing.html). Make sure that the dataset can be traced to a version controlled github repo before this transfer happens because you will need the version tag for the directory structure and nomenclature. An example of the transfer is shown here:
+```
+python /opt/campaigns/hepmc3/scripts/register_to_rucio.py \
```

Comment:
Use an example that will always work because people will run this. Need access to file to upload. Need to avoid did clashes. Prefer to just upload a small file to a date-stamped temporary did 'directory'.